www/js/tpl/server-setup.hbs

```diff
@@ -23,6 +23,10 @@
                             <input type="text" class="form-control" name="hostname" />
                             <p class="help-block"><span class="label label-warning">Hostname cannot be changed after setup</span></p>
                         </div>
+
+                        <div class="checkbox">
+                            <label><input class="delegate-dataset" name="encryption_enabled" type="checkbox">Encryption Enabled</label>
```

Comment:
Is class="delegate-dataset" the right thing - maybe drop that css class name, or I would think we'd have a more generic CSS class name.

Reply:
Yeah, removed.